ibis/file/csv.py

```diff
@@ -26,14 +62,16 @@ class CSVTable(ops.DatabaseTable):
 
 
 class CSVClient(FileClient):
+
+    dialect = PandasDialect
     extension = 'csv'
 
     def insert(self, path, expr, index=False, **kwargs):
         path = self.root / path
         data = execute(expr)
         data.to_csv(str(path), index=index, **kwargs)
 
-    def table(self, name, path=None):
+    def table(self, name, path=None, schema=None):
```

Comment:
I introduced a `schema` argument to allow users to specify a schema for a table if the backend isn't able to infer it.